DNS/DNS_Cache.c

```diff
@@ -226,7 +231,7 @@
         int x;
```

Comment:
Also, since the return type is BaseType, can we make the int to a BaseType? (on this line and the line above).
Thanks